failed-builds-notification.js

```diff
@@ -151,7 +151,7 @@ function sendEmail(context, culprits, currentCulprit, buildUrl, pipelineName) {
 
     const content = `Greetings ${name}!
 
-What in the Shire is this?
+<img src="https://i.imgur.com/HjtirHY.png" width="250px" alt="What in the shire is this?">
```

Comment:
Maybe upload the image to this repo & use the `raw.github.com` URL instead? here's the image

![wildflower](https://user-images.githubusercontent.com/1861842/39860882-477e4944-5482-11e8-9264-43e48621a610.png)